nautobot/extras/api/nested_serializers.py

```diff
@@ -173,13 +173,24 @@ def validate(self, data):
             if "name" not in data:
                 raise serializers.ValidationError({"name": "Please provide a name for the job schedule."})
 
-            if "start_time" not in data or data["start_time"] < models.ScheduledJob.earliest_possible_time():
+            if ("start_time" not in data and data["interval"] != choices.JobExecutionType.TYPE_CUSTOM) or (
+                "start_time" in data and data["start_time"] < models.ScheduledJob.earliest_possible_time()
+            ):
                 raise serializers.ValidationError(
                     {
                         "start_time": "Please enter a valid date and time greater than or equal to the current date and time."
                     }
                 )
 
+            if data["interval"] == choices.JobExecutionType.TYPE_CUSTOM:
+
+                if data.get("crontab") is None:
+                    raise serializers.ValidationError({"crontab": "ya need sth"})
```

Comment:
"ya need sth"? What is that! 😁

Reply:
LOL i put TODOs on stupid comments and debugs so they wouldn't commit but i guess i missed one